client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.html

```diff
@@ -37,34 +37,27 @@
             </button>
         </div>
 
-
-        <div class="description table-responsive mt-2" data-ng-show="DeviceCtrl.showDevices">
-            <table class="table table-hover table-condensed">
-                <thead>
-                <tr>
-                    <th scope="col"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_mac"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_ip"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_host"></th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr data-ng-repeat="device in DeviceCtrl.devices track by device.macAddress">
-                    <td>
-                        <span class="ovh-font h5"
-                           data-ng-class="DeviceCtrl.getConnectionIcon(device)"
-                           title="{{device.interfaceType}}"></span>
-                        <span class="ovh-font h5"
-                           data-ng-class="DeviceCtrl.getDeviceInfo(device).icon"
-                           title="{{DeviceCtrl.getDeviceInfo(device).tooltip | translate}}"></span>
-                    </td>
-                    <th scope="row" data-translate-attr="{ title : 'xdsl_modem_device_mac' }" data-ng-bind="device.macAddress"></th>
-                    <td data-translate-attr="{ title : 'xdsl_modem_device_ip' }" data-ng-bind="device.ipAddress"></td>
-                    <td data-translate-attr="{ title : 'xdsl_modem_device_host' }" data-ng-bind="device.hostName"></td>
-                </tr>
-                </tbody>
-            </table>
+        <div class="description mt-2">
+            <oui-datagrid data-rows="DeviceCtrl.devices">
+                <oui-column data-title="'common_table_header_type' | translate"
+                            data-property="interfaceType" data-sortable="asc">
+                    <span class="ovh-font size-x-large"
```

Comment:
Can't we use available size of `ovh-manager-webfont` instead of creating a new class ?
Take care to the indentation of attributes too ;-)